BryantUniversity/BryantUniversity/Controllers/HomeController.cs

```diff
@@ -1,12 +1,44 @@
-﻿using System.Web.Mvc;
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
 
 namespace BryantUniversity.Controllers
 {
     [AllowAnonymous]
     public class HomeController : Controller
     {
-        // GET: Home
-        public ActionResult Index()
+        private Context context = new Context();
+        [AllowAnonymous]
```

Comment:
HomeController already has the AllowAnonymous attribute. Do you plan to require authentication for some of this controller's actions in the future?